core/src/util/registry.rs

```diff
@@ -0,0 +1,114 @@
+#[cfg(target_os = "windows")]
+pub mod windows {
```

Comment:
I don't think we need this module here, unless we want to change this to have non windows as a module in this file. In that case we'd want to take the cfg out of the mod file above. I'm fine with going either direction, but let's choose one.